tools/data/build_file_list.py

```diff
@@ -147,6 +154,37 @@ def build_list(split):
     return (train_rgb_list, test_rgb_list), (train_flow_list, test_flow_list)
 
 
+def lines2dictlist(lines, format):
+    """Convert lines in 'txt' format to dictions in 'json' format. Currently
+    support single-label and multi-label.
+
+    Args:
+        lines (list): List of lines in 'txt' label format.
+            'frame_dir num_frame label' (rawframes + single-label)
+            'frame_dir num_frame label1 label2 ...' (rawframes + multi-label)
+            'filename label' (videos + single-label)
+            'filename label1 label2 ...' (videos + multi-label)
+        format (str): Data format, choices are 'rawframes' and 'videos'.
+
+    Returns:
+        list: list of dictions
+    """
+    lines = [x.split() for x in lines]
+    if format == 'rawframes':
+        data = [
+            dict(
+                frame_dir=line[0],
+                total_frames=int(line[1]),
+                label=[int(x) for x in line[2:]]) for line in lines
+        ]
+    elif format == 'videos':
+        data = [
+            dict(filename=line[0], label=[int(x) for x in line[1:]])
+            for line in lines
+        ]
```

Comment:
```
else:
    raise ValueError(f'{format} is unsupported')
```

Reply:
Think no need to do so. Since in argparse we already restrict the choices of format to be rawframes and videos